module/zfs/mmp.c

```diff
@@ -557,6 +549,16 @@ mmp_thread(void *arg)
 
 	mmp_thread_enter(mmp, &cpr);
 
+	/*
```

Comment:
nit: Maybe mention in this comment something about why the last write time is set here instead of elsewhere?

Reply:
OK, comment updated.